lib/reporters/dot.js

```diff
@@ -19,9 +19,13 @@ exports = module.exports = Dot;
  *
  * @api public
  * @param {Runner} runner
+ * @param {Object} options
```

Comment:
```js
/*
 * @param {Object} [options] - options
 * @param {Object} [options.reporterOptions] - reporter options
 * @param {boolean} [options.reporterOptions.showErrorsImmediately=false] - if truthy, errors will be 
 *   shown immediately after tests fail instead of after all tests have finished.
 */
```

`boolean` type makes it more clear that this is a *flag*.

Reply:
@boneskull it's not possible for it to be boolean if it's coming from CLI though is it?  I'm assuming any values from `--reporter-options` will be `string`s.

Reply:
hmm, you're probably right.  not a big deal.  this is javascript after all